docs/source/graph-construction/index.rst

```diff
@@ -0,0 +1,18 @@
+==============================
+Distributed Graph Construction
+==============================
+
```

Comment:
Please give a short intro of what is GSProcessing.
You can move some of the content from `GraphStorm Processing Prerequisites` to here.
@jalencato @thvasilo @zhjwy9343

Reply:
Done for that